identity-wallet/src/lib.rs

```diff
@@ -6,11 +6,7 @@ pub mod utils;
 pub mod verifiable_credential_record;
 
 // Re-exports
-pub use oid4vc_core;
-pub use oid4vc_manager;
-pub use oid4vci;
-pub use oid4vp;
-pub use siopv2;
+pub use oid4vc::*;
```

Comment:
I think we should generally avoid wildcard imports.

Reply:
Well its a reexport of a reexport but I will change it